modules/doc/content/application_development/build_system.md

```diff
@@ -208,3 +208,49 @@ when building MOOSE or a MOOSE-based application.
 
 - `framework/conf_var.mk.in` - This file is where you put your new expansion expressions. Typically these
   are in the form of "@VARAIBLE@".
+
+### Make Install
+
+MOOSE's build system has the ability to create installed binaries suitable for use with the conda package manager, or installation on a shared computing resource such as Sawtooth. The `install` target will copy binary and required libraries to a file tree based on the prefix you configured with (set by the variable `PREFIX` or the `--prefix` argument.
+
+```
+$ cd moose/framework
+$ ./configure --prefix=<installation path>
+$ make
+$ make install
+```
+
+Additionally each application may wish to make an one or more file structures available for end users to install into their own environment. This allows for end-users to install and run the application test suite, examples, tutorials, or other inputs of interest. To customize what is available for installation, the application developer will define the list of directories in their application make file. For example:
+
+```
+INSTALLABLE_DIRS := test/tests examples
```

Comment:
What does this mean? Does this copy `test/tests` into `<installation path>/examples`?

Reply:
And `examples` is considered as a type? How if I have something like
```
INSTALLABLE_DIRS += test/foo examples/foo
```
?

Reply:
You may be overthinking this. This is simply the list of directories that will be installable. It's not a "To" or "From", or a "type". 
In the above example, users would be able to install "tests" and "examples". There is one point of clarification though and I can see now that there is a potential deficiency in the way I'm currently handling it. If you supply a path with slashes in it, I'm currently only taking the last directory as the directory a user will type in the `--copy-inputs` command (so "tests" or "examples"). If you have more complicated structures like "tests/installable examples/installable problems/installable", it's not going to work as it's coded now. 

However, I don't want people to end up with "test_tests" or other complicated things to remember so I didn't try to automatically flatten these structures... I might leave it this way for right now and if needed code up a way for developers to give their installed directories.